.github/workflows/go.yml

```diff
@@ -9,6 +9,14 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      PORT: ${{secrets.PORT}}
+      READ_HEADER_TIMEOUT: ${{secrets.READ_HEADER_TIMEOUT}}
```

Comment:
I didn't say to put all environment into "github secrets", but only private information - like db login/password